packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -625,6 +626,7 @@ export const SpacePlugin = ({
 
               if (intent.data?.target instanceof SpaceProxy) {
                 const space = intent.data.target;
+                await space.waitUntilReady();
```

Comment:
Under what circumstances would we be adding an object to a space that's not open yet?

Reply:
Valid point, I can delete this, but it is just a sanity check